modules/@apostrophecms/home-page/views/page.html

```diff
@@ -9,9 +9,13 @@
 {% block sideNav %}{% endblock %}
 
 {% block main %}
-  <section class="latest-article">
-    <h2>Latest Article</h2>
-    {% component "article:recent" with { limit: 1 } %}
-  </section>
-  {% area data.page, 'main' %}
+  <div class="home">
+    <section class="latest-article">
+      <h2>Latest Articles</h2>
+      {% component "article:recent" with { limit: 5 } %}
```

Comment:
All the ❤️ for leaving in this demo of probably the nicest server side feature of 3.x.